reledpar.dtx

```diff
@@ -4446,7 +4446,7 @@
   \global\advance\edtext@later by \@ne%
 %    \end{macrocode}
 %    \begin{macrocode}
-% As we are in a simili-\cs{edtext}, we need no to get the \cs{sameword} data stored in the auxiliary file for this \cs{edtextlater}.
+% As we are in a simili-\cs{edtext}, we need not get the \cs{sameword} data stored in the auxiliary file for this \cs{edtextlater}. %Please note that I'm not sure what is meant by "simili-".
```

Comment:
simili is a french expression from Latin. Apparently it does not exist on English. "no" was for "now".